src/Context/EventContext.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState, useEffect, createContext } from 'react'
+import apiCalls from '../utilities/apiCalls';
+
+const EventContext = createContext()
+
+const EventProvider = props => {
```

Comment:
Context looks super readable. I appreciate you taking the time to implement this for us, and seemingly so easily. I look forward to working with Context going forward.